app/src/fyp-app/fyp-app.ts

```diff
@@ -143,7 +143,7 @@ class FypApp extends RuntimeComponent {
         const ed = this.getShadowRoot().getElementById("editor");
         if(ed)
             return ed as FYPEditor;
-        throw new Error("Theip air an éagarthóir a fháil");
+        throw new Error("Theip air an eagarthóir a fháil");
```

Comment:
Níl mé sásta go bhfuil an teachtaireacht seo ceart. As Béarla déarfainn "Failed to get the editor" nó rud mar sin.

Reply:
Measaim go mbeadh "Fáil eagarthóir teipthe" oiriúnach

Reply:
D'athraigh mé gach teachtaireacht earráide mar sin